lib/buffer.js

```diff
@@ -22,19 +22,16 @@
 'use strict';
 
 const binding = process.binding('buffer');
+const config = process.binding('config');
 const { compare: compare_, compareOffset } = binding;
 const { isArrayBuffer, isSharedArrayBuffer, isUint8Array } =
     process.binding('util');
 const bindingObj = {};
 const internalUtil = require('internal/util');
+const pendingDeprecation = !!config.pendingDeprecation;
 
-class FastBuffer extends Uint8Array {
-  constructor(arg1, arg2, arg3) {
-    super(arg1, arg2, arg3);
-  }
-}
+const FastBuffer = (() => class Buffer extends Uint8Array {})();
```

Comment:
@jasnell Do you have that code pushed somewhere? I mean, if it does have no effect, that would be a bug, right?